roles/gluster_hci/tasks/hci_volumes.yml

```diff
@@ -1,5 +1,21 @@
 ---
 # Create volumes for gluster_hci
+
+- name: check if gluster_features_hci_cluster is set
+  fail:
+    msg: "gluster_features_hci_cluster is undefined"
+  when: gluster_features_hci_cluster is not defined
+
+- name: check if gluster_features_hci_volume_options is set
+  fail:
+    msg: "gluster_features_hci_volume_options is undefined"
+  when: gluster_features_hci_volume_options is not defined
+
+- name: check if gluster_features_hci_volumes is set
+  fail:
+    msg: "gluster_features_hci_volumes is undefined"
```

Comment:
Can you change the message to:

Variable gluster_features_hci_volumes has to be defined.